express_relay/sdk/js/src/index.ts

```diff
@@ -303,6 +318,35 @@ export class Client {
     }
   }
 
+  getPermittedTokens(
+    sellTokens: TokenAmount[],
+    bidAmount: bigint,
+    value: bigint,
+    wethAddress: Address
+  ) {
+    const permittedTokens = sellTokens.map((token) => ({
+      ...token,
+    }));
+    const wethAmount = bidAmount + value;
```

Comment:
weth can also be a sellToken, so this may be better named `wethAmountBidValue` or something to distinguish it from weth in the sellTokens